src/App.jsx

```diff
@@ -0,0 +1,20 @@
+
+import './App.css';
+import { Routes, Route } from 'react-router';
+import Header from './components/Header';
+import React from 'react';
+import Books from './screens/Books';
+
+
+function App() {
+  return (
+    <main className='w-full h-screen bg-slate-200'>
+      <Header />
+      <Routes>
+        <Route element={<Books />} path='/' />
+      </Routes>
+    </main>
```

Comment:
Kindly note that in the project requirement, you were asked to add two routes, for this two components 

Project requirement :

![image](https://github.com/DjibrilM/book-store/assets/20537712/d90aeb81-e752-4c61-93b0-7ca307b802b8)

Reply:
well resolved! ✔️